java/ql/src/Security/CWE/CWE-319/HttpsUrls.ql

```diff
@@ -63,6 +50,13 @@ class HTTPStringToURLOpenMethodFlowConfig extends TaintTracking::Configuration {
   }
 }
 
+/**
+ * A sink that represents a URL opening method call, such as a call to `java.net.URL.openConnection()`.
+ */
+private class URLOpenSink extends DataFlow::Node {
```

Comment:
```suggestion
private class UrlOpenSink extends DataFlow::Node {
```

Reply:
I don't mind changing this to `UrlOpenSink`, but a couple of lines above we already have `HTTPStringToURLOpenMethodFlowConfig`, so it would look a bit strange, wouldn't it?

Reply:
Right, that ought to be renamed to `HttpStringToUrlOpenMethodFlowConfig`.